scripts/t265_to_mavlink.py

```diff
@@ -580,6 +583,10 @@ def user_input_monitor():
                     if (position_displacement > jump_threshold):
                         send_msg_to_gcs('Pose jump detected')
                         print("Position jumped by: ", position_displacement)
+
+                        if reset_counter >= 255:
+                            reset_counter = 1
+
                         reset_counter += 1
```

Comment:
Same as above, the check can be put after the increment.
Additionally, the extra empty lines can be removed.

Reply:
@thien94 sorry for that, I have removed the empty line, I have test the way you said yesterday, because MAVLink Pack and increment is running on different task, so it will still raise errors and the pack is exec between increment and check

Reply:
I see. In my test that never happens but I guess it can happen in some rare cases or when the increment happens frequent enough.

In that case, you can include the check in the vision_position_message (like before) as well as the vision_speed_estimate message: https://github.com/thien94/vision_to_mavros/blob/master/scripts/t265_to_mavlink.py#L306. These are the messages that consume the `reset_counter` at the moment.